### PR TITLE
Refs #20394 - Send the volume parameters in template

### PR DESCRIPTION
**Pull request by shiramax**

### CODE REVIEW

File: lib/fog/ovirt/requests/compute/v4/list_template_volumes.rb [hunk 1]
```diff
@@ -3,15 +3,22 @@ module Compute
...
-
             attachments.map do |attachment|
-              ovirt_attrs client.follow_link(attachment.disk)
+              attachment_disk = client.follow_link(attachment.disk)
+              attachment = client.follow_link(attachment)
+              bootable = attachment.bootable
+              interface = attachment.interface
+              attachment_disk.bootable = bootable if attachment_disk.bootable.nil?
+              attachment_disk.interface = interface if attachment_disk.interface.nil?
+              attachment_disk.storage_domain = attachment_disk.storage_domains[0].id
```
Comment (orrabin) [inline lib/fog/ovirt/requests/compute/v4/list_template_volumes.rb]: This is probably not possible but I would suggest checking there is something in storage_domains before trying to take the first one.

**Review by orrabin — CHANGES_REQUESTED** [review]
Works well, I have 1 nitpick, after that this can be merged.

**Comment by shiramax** [comment]
Thanks @orrabin , please review again :)

**Review by orrabin — APPROVED** [review]
The UI now shows the storage domain of the disk + the first on the list.
This is consistent with v3 that has the same bug. It's fixed after save and it's an issue in foreman UI.

**Comment by orrabin** [comment]
Thanks @shiramax !

**Pull request merged**